data-prepper-plugins/http-source/src/test/java/com/amazon/dataprepper/plugins/source/loghttp/HTTPSourceTest.java

```diff
@@ -263,6 +270,32 @@ public void testHealthCheck() {
                 .whenComplete((i, ex) -> assertSecureResponseWithStatusCode(i, HttpStatus.OK)).join();
     }
 
+    @Test
+    public void testHealthCheckUnauthenticatedDisabled() {
+        // Prepare
+        lenient().when(sourceConfig.isUnauthenticatedHealthCheck()).thenReturn(false);
```

Comment:
Is lenient required in this test? Usually lenient mocks are used in a `@BeforeEach` or `@BeforeAll`

Reply:
I was following how the other `when()`s where being produced. I'll remove it if it isn't necessary